docs/cristin.ttl

```diff
@@ -16,35 +16,55 @@
 :Published
     a               rdfs:Class ;
     rdfs:label      "PUBLISHED" ;
+    rdfs:label      "PUBLISERT" ;
     rdfs:subClassOf :PublicationStatus .
 
+:InPress
+    a               rdfs:Class ;
+    rdfs:label      "IN_PRESS" ;
+    rdfs:subClassOf :PublicationStatus .
+
+::TestPublicationStatus
+    a               rdfs:Class ;
+    rdfs:label      "TEST" ;
+    rdfs:subClassOf :PublicationStatus .
+
+
 :VARBEID_SPRAK_Class
     a            rdfs:Class ;
     rdfs:comment "The structure defining the VARBEID_SPRAK property." .
 
 :VARBEID_PERSON_Class
     a            rdfs:Class ;
     rdfs:comment """
-    The structure of the VARARBEID_PERSON entry. It contains information about the status of the contributor
-    at the time the Result was registered in Cristin.
+    The structure of the VARARBEID_PERSON entry. It contains user submitted information about the status of the contributor
+    at the time the Result was registered in Cristin. Some of the information is provided by the user manually in order to reflect the
```

Comment:
"Some information is provided ... manually" - I'd go for "may be", because it defaults to what the person's name is in the person registry at the time of result registration.